cmd/junitreport/pkg/api/test_case.go

```diff
@@ -0,0 +1,38 @@
+package api
+
+import (
+	"strings"
+	"time"
+)
+
+// SetDuration sets the runtime duration of the test case
```

Comment:
Because I don't really know what I'm doing and these started out without methods like in Origin. Should I just collapse it all into one file?